src/app/(pages)/students/page.tsx

```diff
@@ -1,5 +1,47 @@
-import React from "react";
+import Section, { TeamDataType } from "./_components/Section"
+import getStudents from "@/lib/notion/students"
+import { v4 as uuidv4 } from "uuid"
 
-export default function Students() {
-  return <div>Students page</div>;
+export default async function Students() {
+  const students = await getStudents();
+
+  let teamNamesSet = new Set<string>();
+  for(const student of students){
+    teamNamesSet.add(student.team)
+  }
+
+  const teamNamesArray = Array.from(teamNamesSet)
+  const teams: TeamDataType[] = []
+  for(const teamName of teamNamesArray){
+    const teamMembers = students.filter(student => (student.team == teamName))
+    teams.push({teamName, teamMembers})
+  }
+
+  return (
+    <div>
+      <div className=" min-h-screen overflow-x-hidden">
+        <div className="flex flex-col w-full space-y-24 relative pb-24">
+          <section className="content container">
+            <div className="flex flex-col space-y-2">
+              <div className="mb-8">
+                <h2 className="text-4xl md:text-6xl font-extrabold mb-8 text-center pt-6">
+                  Meet the Team
+                </h2>
+                <p className="text-lg text-center">
```

Comment:
Add a width styling to make the text a bit more compressed. I would say set it to `w-1/2` and you will probably need to add some styling to the containing `<div>` to center it, this should work: `flex flex-col items-center`